src/rpc/misc.cpp

```diff
@@ -71,7 +71,7 @@ UniValue getinfo(const JSONRPCRequest& request)
             "  \"transparentsupply\" : n       (numeric) The sum of the value of all unspent outputs when the chainstate was\n"
             "                                            last flushed to disk (use getsupplyinfo to know the update-height, or\n"
             "                                            to trigger the money supply update/recalculation)"
-            "  \"shieldedsupply\": n           (numeric) Chain tip shielded pool value\n"
+            "  \"shieldsupply\": n             (numeric) Chain tip shield pool value\n"
```

Comment:
we need to also update line 136.
Both the key pushed in the returned object here, and the one of `supply_info`, returned from `getsupplyinfo`, from where we pick the value.

Reply:
Hmm, why `sapling_supply.py` is passing?

Reply:
because it calls directly `getsupplyinfo`. Not `getinfo`.

Reply:
kk, all good then.

Reply:
fixed